CHANGELOG.rst

```diff
@@ -19,19 +23,25 @@ Added
 Changed
 -------
 - bot messages contain the `timestamp` of the `BotUttered` event, which can be used in channels
-
-
-Changed
--------
-- NLU evaluations now also stores its output in the output directory like the core evaluation
+- substitute LSTM with Transformer in ``EmbeddingPolicy``
+- ``EmbeddingPolicy`` can now use ``MaxHistoryTrackerFeaturizer``
+- non zero ``evaluate_on_num_examples`` in ``EmbeddingPolicy``
+  and ``EmbeddingIntentClassifier`` is the size of
+  hold out validation set that is excluded from training data
+- NLU evaluations now also stores its output in the output directory
+  like the core evaluation
+- defaults parameters for both ``EmbeddingPolicy`` and ``EmbeddingIntentClassifier``
+  are changed
 
 Removed
 -------
 
 
 Fixed
 -----
-
+- ``MappingPolicy`` standard featurizer is set to ``None``
+- ``Flood control exceeded`` error in Telegram connector which happened because the
```

Comment:
because I added featurizer tests, because code coverage was complaining

``Flood control exceeded`` error in Telegram connector which happened because the - is not part of this PR, it is CHANGELOG merging bug

Reply:
:+1: